packages/core/package.json

```diff
@@ -0,0 +1,117 @@
+{
+  "name": "@inrupt/solid-client-authn-browser",
```

Comment:
Should the first created package be `package/core` or `package/browser` ? The latter aligne with the current codebase.

Reply:
Yeah, I deliberately left it as `browser` so that the build artifact would still be the same as what we currently have. Once I do the actual split, this will change over to be `-core`.